ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -44,3 +44,43 @@ nginx_sites:
       balancer: "{{ deploy_env }}_formplayer/"
       proxy_next_upstream_tries: 1
       proxy_read_timeout: 900s
+    - name: /files
```

Comment:
yes they were manually added. Is there no way to have them site specific? Could we not put a new section into the icds vars file? Or at least have a `when` condition that prevents them from being added to the other envs?